lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -116,10 +116,36 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     if (fci) v5andv6metrics.push(fci);
     if (fmp) v5andv6metrics.push(fmp);
 
+    /** @type {Record<string, string>} */
+    const acronymMapping = {
+      'cumulative-layout-shift': 'CLS',
+      'first-contentful-paint': 'FCP',
+      'first-cpu-idle': 'FCI',
+      'first-meaningful-paint': 'FMP',
+      'interactive': 'TTI',
+      'largest-contentful-paint': 'LCP',
+      'speed-index': 'SI',
+      'total-blocking-time': 'TBT',
+    };
+
+    /**
+     * Clamp figure to 2 decimal places
+     * @param {number} val
+     * @return {number}
+     */
+    const clampTo2Decimals = val => Math.round(val * 100) / 100;
+
     const metricPairs = v5andv6metrics.map(audit => {
-      const value = typeof audit.result.numericValue !== 'undefined' ?
-        audit.result.numericValue.toString() : 'null';
-      return [audit.id, value];
+      let value;
+      if (typeof audit.result.numericValue === 'number') {
+        value = audit.id === 'cumulative-layout-shift' ?
+          clampTo2Decimals(audit.result.numericValue) :
+          Math.round(audit.result.numericValue);
+        value = String(value);
```

Comment:
```suggestion
        value = value.toString();
```

Reply:
I am super curious where the origin of this preference came from :)

Reply:
instantiation of a new primitive object via the constructor is old and gross.

1997 javascript is full of `x = new Array()` instead of array literals `x = []`

at one point it used more memory to leverage the constuctor.

and our favoritest site for microbenchmarks still thinks the method is faster 

![image](https://user-images.githubusercontent.com/39191/84837913-0ccc1500-afee-11ea-912b-a17a799e7de7.png)

and generally if an object provides its own method to convert to another type, i trust it more than i trust a global constructor.

Reply:
> instantiation of a new primitive object via the constructor is old and gross.

agreed I would never encourage `new String(foo)` or `new Boolean()`   an object instead of the primitive is just plain wrong in many cases 👍

but, IMO, the casts are simply casts that communicate as clearly as possible what's being done in a safe way (never get an undefined TypeError) that's also consistent across `String`/`Boolean`/`Number`. 

in this case those benefits don't really apply so it doesn't really matter to me `toString` v. `String`, but the fact that it mattered enough to comment on for you was interesting to me :)

is it primarily that `String(foo)` is too easily confused for `new String(foo)` when reading?